src/sqlfluff/cli/commands.py

```diff
@@ -401,6 +401,12 @@ def get_linter_and_formatter(
 @click.version_option()
 def cli():
     """SQLFluff is a modular SQL linter for humans."""  # noqa D403
+cli.epilog="""\b\bExamples:\n
```

Comment:
IIUC, '\b` is a backspace character. What's this for?

Also, the Black linter check is failing because there are no spaces around the `=`. Can you fix this, please?

Reply:
Sorry, the tests ran forever, and and I wasn't watching them. I must have missed the linting failure. I've fixed it up, and linting is passing now.

The `\b`s are to dedent the "Examples:" heading, because click doesn't provide a top level way (as far as I can find in the docs) to append documentation after it's done it's auto-generated stuff. There is an `epilog` field to d this on groups, so I used the last (only) group's (`cli`) epilog field to provide the examples. The problem is that click renders it indented to the same level as the group's command/argument help items. I could have added an extra newline, because it only seems to indent the first line, but that looked weird. AFAIK the `\b` is pretty standard (work in *nix shells, and in windows consoles).